File: cpp/insertion_Sort.cpp
void insertion_Sort(int arr[], int n){
    int key,i,j;
    for( i = 1 ; i < n ; i ++){
        key=arr[i];
        j=i-1;
        while(j >= 0 && arr[j] > key){
            arr[j+1]=arr[j];
            j--;
        }
        arr[j+1]=key;
    }
}
//Function to print the array after sorting
void printArray(int arr[],int n){
    for( int i = 0; i < n; i ++){
        cout<<arr[i]<<" ";
    } 
}

File: javascript/insertionSort.js
let insertionSort = (arr) => {
    let i,j,ele;
    for(i = 1;i<arr.length;i++){
        j = i - 1;
        ele = arr[i];
        while(j>=0 && arr[j]>ele){
            a[j+1] = a[j];
            j--;
        }
        a[j+1] = ele;
    }
}
let a = [2,1,6,4,7,5]
insertionSort(a);
console.log(a);